project-keeper/src/main/java/com/exasol/projectkeeper/githuboutput/FileOutputPublisher.java

```diff
@@ -0,0 +1,64 @@
+package com.exasol.projectkeeper.githuboutput;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.*;
+import java.util.logging.Logger;
+
+import com.exasol.errorreporting.ExaError;
+
+// [impl->dsn~verify-modes.output-parameters~1]
+class FileOutputPublisher implements OutputPublisher {
```

Comment:
Maybe rename to `FileContentProvider` then?

Reply:
Although these classes are package-private, I think some comments at least on class level would be beneficial.